packages/emptystate/svgr.config.js

```diff
@@ -0,0 +1,45 @@
+const babelTypes = require('@babel/types')
```

Comment:
this svgr library seems like a more straightforward approach to convert the svg to React components. it was pretty easy to implement